sklearn/impute/_base.py

```diff
@@ -465,11 +465,45 @@ def transform(self, X):
             n_missing = np.sum(mask, axis=0)
             values = np.repeat(valid_statistics, n_missing)
             coordinates = np.where(mask.transpose())[::-1]
-
             X[coordinates] = values
 
         return super()._concatenate_indicator(X, X_indicator)
 
+    def inverse_transform(self, X):
+        """Convert the data back to the original representation.
+
+        Inverts the ``fit_transform`` operation performed on an array.
+        This operation can only be performed after :class:`SimpleImputer` is
+        instantiated with ``add_indicator`` parameter set to `True`.
+
+        Note that ``inverse_transform`` can only regenerate data for the
+        features that have missing indicators present in the fitting data.
+        This limitation exists because the imputer learns missing indicators
+        based on features of fitting-data, and they are only set during
+        the ``fit`` phase.
+
+        Parameters
+        ----------
+        X : array-like,
```

Comment:
You should add a backslash to have the shape rendered in the same line in the website.
```suggestion
        X : array-like, \
```